pdns/lua-record.cc

```diff
@@ -126,9 +128,20 @@ class IsUpOracle
         throw std::runtime_error(boost::str(boost::format("unable to match content with `%s`") % cd.opts.at("stringmatch")));
       }
 
-      if(!status) {
-        g_log<<Logger::Info<<"LUA record monitoring declaring "<<remstring<<" UP for URL "<<cd.url<<"!"<<endl;
+      int weight = 0;
+      try {
+        weight = stoi(content);
```

Comment:
so this tries to parse the content as a number -always- and then stores it in weight if it parses? interesting. seems harmless :)